src/features/Home/HeroHeader/HeroHeader.module.scss

```diff
@@ -1,37 +1,37 @@
 @use 'src/styles/utility' as *;
 .HeroImageStyle {
-    background-color: var(--brown-02);
-    background-image: linear-gradient(rgba(0, 0, 0, 0.16), rgba(0, 0, 0, 0.16)), url(/img/header.png);
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 100%;
-    height: rem(60);
+  background-color: var(--brown-02);
```

Comment:
I think this is still old CSS variable from previous project. Replace it maybe with `--ifm-color-secondary-contrast-foreground`